frontend/src/components/molecules/MatchTeam/MatchTeam.tsx

```diff
@@ -39,32 +39,39 @@ const useStyles = makeStyles((theme) => ({
     inline: {
         display: 'inline',
     },
+
+    waitingForTeam: {
+        display: 'inline',
+        color: 'red'
+    },
+
 }));
 
 export const MatchTeam = (props: MatchTeamProps) => {
     const classes = useStyles();
 
-    const noNumber = "NN";
+    const noInformation = "Brak danych";
+    const WaitingForTeam = <span className={classes.waitingForTeam}>"Oczekiwanie"</span>;
     const player1 = props.player1 || "player1";
     const player2 = props.player2 || "player2";
     const avatarSymbol = (typeof props.teamNumber === "string" ? props.teamNumber[0] : props.teamNumber)
-    const teamNumber = avatarSymbol || noNumber;
-    const currentPlayerLevel = (props.currentPlayerLevel || props.currentPlayerLevel === 0) ? props.currentPlayerLevel : noNumber;
+    const teamName = props.teamNumber;
```

Comment:
Raczej w ogole nie kojarze czegos takiego jak teamName. Zazwyczaj to po prostu imie i naziwsko obu graczy :) @lamparina ?

Reply:
ok :) 
Dobrze Aniu, że jesteś i piszesz. Dobrze znać taką specjalisytczną wiedzę - nawet z piłkarzyków!

Reply:
ok